security/pkg/server/ca/server_test.go

```diff
@@ -70,6 +76,130 @@ func (authn *mockAuthenticator) Authenticate(ctx context.Context) (*authenticate
 	}, nil
 }
 
+type mockAuthInfo struct {
+	authType string
+}
+
+func (ai mockAuthInfo) AuthType() string {
+	return ai.authType
+}
+
+func TestCreateCertificateE2EWithCertificates(t *testing.T) {
```

Comment:
Could you add a comment above the test about test? Emphasize that this is testing against the client cert authenticator, to differentiate the mock authenticator used in the other test.